packages/ee/lib/service.ts

```diff
@@ -1,13 +1,109 @@
 import "server-only";
 
+import { cache, revalidateTag } from "@formbricks/lib/cache";
 import { ENTERPRISE_LICENSE_KEY, IS_FORMBRICKS_CLOUD } from "@formbricks/lib/constants";
+import { hashString } from "@formbricks/lib/hashString";
 import { TTeam } from "@formbricks/types/teams";
 
-export const getIsEnterpriseEdition = (): boolean => {
-  if (ENTERPRISE_LICENSE_KEY) {
-    return ENTERPRISE_LICENSE_KEY.length > 0;
+import { prisma } from "../../database/src";
+
+const getPreviousResult = (): Promise<{ active: boolean | null; lastChecked: Date }> =>
```

Comment:
Can you please add some comments to explain what these functions do and why they are needed